content/blog/2018-03-27-update-on-async-rendering.md

```diff
@@ -23,7 +23,7 @@ These lifecycle methods have often been misunderstood and subtly misused; furthe
 
 **Note that if you're an application developer, you don't have to do anything about the legacy methods yet. The primary purpose of the upcoming version 16.3 release is to enable open source project maintainers to update their libraries in advance of any deprecation warnings. Those warnings will not be enabled until a future 16.x release.**
 
-At Facebook, we maintain over 50,000 React components, so we're in the same boat as you. We can't rewrite our apps so we will take the gradual migration path together with everyone in the React community.
+We maintain over 50,000 React components at Facebook, so we understand that migrations take time. We will take the gradual migration path together with everyone in the React community.
```

Comment:
I think it's important to point out our migration path isn't "tell every product team to set everything aside and start fixing their components". People commonly assume that we have this kind of power at Facebook and that product teams just go ahead and spend months catching up with our recommendations (and a "small company wouldn't be able to do this"). So I would prefer if we were very clear in our wording that this is not the case.

Reply:
Hm. Gotcha.

I don't think the previous wording, "_we can't rewrite our apps_", conveyed that. (It didn't really make sense to me.)

Reply:
Sure—you're a native speaker so I was just trying to give you the context into what I was trying to express.

Reply:
Sorry, I wasn't trying to criticize the wording or say it didn't sound native. I just don't think it conveyed the meaning you are wanting to convey.

Reply:
How about:
> We maintain over 50,000 React components at Facebook, and we don’t plan to rewrite them all immediately. We understand that migrations take time. We will take the gradual migration path along with everyone in the React community.

Reply:
Nice, I like it